integration_tests/testsuite/test_util.py

```diff
@@ -54,10 +64,15 @@ def _generate_int64_token():
     return random.randint(-(2 ** 31), (2 ** 31)-1)
 
 
-def generate_logging_payload():
-    data = {'log_name': _generate_name(),
-            'token': _generate_hex_token()}
-    return data
+def generate_logging_payloads():
+    payloads = []
+    for l in list(Severity):
```

Comment:
yeah you're right. I was originally trying to use the number code instead of the string, but I'll just stick to using the name.